espn_api/basketball/league.py

```diff
@@ -100,7 +100,7 @@ def recent_activity(self, size: int = 25, msg_type: str = None, offset: int = 0)
         headers = {'x-fantasy-filter': json.dumps(filters)}
         data = self.espn_request.league_get(extend='/communication/', params=params, headers=headers)
         data = data['topics']
-        activity = [Activity(topic, self.player_map, self.get_team_data) for topic in data]
+        activity = [act for act in (Activity(topic, self.player_map, self.get_team_data, include_moved=include_moved) for topic in data)if act and str(act).strip()]
```

Comment:
Why did this change other than adding `include_moved=include_moved`?

Reply:
I don't remember what I was trying to do here, changed back to the original and added the include_moved=include_moved